{"uid":"1db1a0c30c5b781d","name":"测试错误的邀请码","fullName":"tests.test_register.TestRegister#test_register_wrong_invitation_code","historyId":"2ba0d6ba5e9f8e162cee8dc8c426008a","time":{"start":1746376650427,"stop":1746376651353,"duration":926},"description":"测试错误的邀请码","descriptionHtml":"<p>测试错误的邀请码</p>\n","status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>> = <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>.is_registration_successful","statusTrace":"self = <tests.test_register.TestRegister object at 0x000002C0B75C4810>\ndriver = <utils.webdriver.WebDriver object at 0x000002C0B7539910>\nbase_url = 'http://211.87.232.162:8080'\n\n    @allure.story(\"错误邀请码测试\")\n    @allure.title(\"测试错误的邀请码\")\n    def test_register_wrong_invitation_code(self, driver, base_url):\n        \"\"\"测试错误的邀请码\"\"\"\n        with allure.step(\"创建注册页面对象\"):\n            register_page = RegisterPage(driver)\n    \n        with allure.step(f\"导航到注册页面 {base_url}\"):\n            register_page.navigate_to(base_url)\n            register_page.wait_for_page_load()\n    \n        with allure.step(\"生成测试数据\"):\n            username = self.generate_random_username()\n    \n        with allure.step(\"填写表单（错误的邀请码）\"):\n            register_page.set_username(username)\n            register_page.set_password(\"Test@123\")\n            register_page.set_confirm_password(\"Test@123\")\n            register_page.set_invitation_code(\"9999\")  # 错误的邀请码\n            register_page.click_register()\n    \n        with allure.step(\"验证注册失败\"):\n>           assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\nE           AssertionError: 使用错误的邀请码却仍然注册成功\nE           assert not True\nE            +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>>()\nE            +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>> = <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>.is_registration_successful\n\ntests\\test_register.py:139: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"web_driver","time":{"start":1746376606452,"stop":1746376609998,"duration":3546},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"driver","time":{"start":1746376650426,"stop":1746376650426,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"base_url","time":{"start":1746376650426,"stop":1746376650426,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"testStage":{"description":"测试错误的邀请码","status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>> = <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>.is_registration_successful","statusTrace":"self = <tests.test_register.TestRegister object at 0x000002C0B75C4810>\ndriver = <utils.webdriver.WebDriver object at 0x000002C0B7539910>\nbase_url = 'http://211.87.232.162:8080'\n\n    @allure.story(\"错误邀请码测试\")\n    @allure.title(\"测试错误的邀请码\")\n    def test_register_wrong_invitation_code(self, driver, base_url):\n        \"\"\"测试错误的邀请码\"\"\"\n        with allure.step(\"创建注册页面对象\"):\n            register_page = RegisterPage(driver)\n    \n        with allure.step(f\"导航到注册页面 {base_url}\"):\n            register_page.navigate_to(base_url)\n            register_page.wait_for_page_load()\n    \n        with allure.step(\"生成测试数据\"):\n            username = self.generate_random_username()\n    \n        with allure.step(\"填写表单（错误的邀请码）\"):\n            register_page.set_username(username)\n            register_page.set_password(\"Test@123\")\n            register_page.set_confirm_password(\"Test@123\")\n            register_page.set_invitation_code(\"9999\")  # 错误的邀请码\n            register_page.click_register()\n    \n        with allure.step(\"验证注册失败\"):\n>           assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\nE           AssertionError: 使用错误的邀请码却仍然注册成功\nE           assert not True\nE            +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>>()\nE            +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>> = <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>.is_registration_successful\n\ntests\\test_register.py:139: AssertionError","steps":[{"name":"创建注册页面对象","time":{"start":1746376650427,"stop":1746376650427,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"导航到注册页面 http://211.87.232.162:8080","time":{"start":1746376650427,"stop":1746376650573,"duration":146},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"生成测试数据","time":{"start":1746376650573,"stop":1746376650573,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"填写表单（错误的邀请码）","time":{"start":1746376650573,"stop":1746376651345,"duration":772},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"验证注册失败","time":{"start":1746376651345,"stop":1746376651352,"duration":7},"status":"failed","statusMessage":"AssertionError: 使用错误的邀请码却仍然注册成功\nassert not True\n +  where True = <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>>()\n +    where <bound method RegisterPage.is_registration_successful of <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>> = <pages.register_page.RegisterPage object at 0x000002C0B76A7ED0>.is_registration_successful\n","statusTrace":"  File \"D:\\学习资料\\软件测试\\test2\\tests\\test_register.py\", line 139, in test_register_wrong_invitation_code\n    assert not register_page.is_registration_successful(), \"使用错误的邀请码却仍然注册成功\"\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":0}],"attachments":[{"uid":"b383094ecd614ea8","name":"失败截图_20250505_003731","source":"b383094ecd614ea8.png","type":"image/png","size":25751},{"uid":"ea4297687063c119","name":"页面源代码","source":"ea4297687063c119.html","type":"text/html","size":1163},{"uid":"8af8250ddcbb3bb1","name":"stdout","source":"8af8250ddcbb3bb1.txt","type":"text/plain","size":397}],"parameters":[],"hasContent":true,"stepsCount":5,"attachmentStep":false,"shouldDisplayMessage":true,"attachmentsCount":3},"afterStages":[{"name":"web_driver::0","time":{"start":1746376686247,"stop":1746376688400,"duration":2153},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"stepsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"labels":[{"name":"epic","value":"用户管理"},{"name":"feature","value":"注册功能"},{"name":"story","value":"错误邀请码测试"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_register"},{"name":"subSuite","value":"TestRegister"},{"name":"host","value":"djc"},{"name":"thread","value":"15068-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_register"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"1db1a0c30c5b781d.json","parameterValues":[]}